.env.example

```diff
@@ -2,3 +2,4 @@ DAPI_SEED=
 FAUCET_PRIVATE_KEY=
 NETWORK=
 DPNS_CONTRACT_ID=
+
```

Comment:
?

Reply:
`chore: useless commit`
because I didn't want to mangle with `amend`, otherwise semantic PR check won't pass